openapi/openapi.yml

```diff
@@ -1767,6 +1767,40 @@ paths:
           $ref: '#/components/responses/Error'
       tags:
         - Postgres Database
+  '/project/{project_id}/postgres/locations':
```

Comment:
I think the name of the endpoint (`locations`) does not reflect well on what this endpoint actually returns. It doesn't just return available locations, but also available VM sizes and PostgreSQL versions.

How about something like `capabilities` or `options`? Latter is bit too generic, but `capabilities` seem to capture the essence of what we are trying to do here.